lib/node_modules/@stdlib/repl/lib/main.js

```diff
@@ -241,6 +242,9 @@ function REPL( options ) {
 	setNonEnumerable( this, '_multiline', {} );
```

Comment:
Would it make more sense to move this state to the multi-line class and then provide public APIs that the REPL instance can query? Not clear to me that, with this PR, the state is held at the right level of abstraction.

Reply:
Currently `process_line.js` also uses the multiline object using `repl._multiline`. After refactoring that would be `repl._multilineHandler.variable` or should I pass in the multiHandler instance as an argument to `process_line` if that makes accessing multilineHandler class variables easier?

Reply:
Yeah, that seems reasonable. `multilineHandler` can provide accessor APIs for querying its state. `processLine` can then use those public APIs, just as the REPL instance does.

Reply:
I was thinking, what if I move `processLine` inside the `multilineHandler` class. So it will be something like `multilineHandler.processLine()` ? And also maybe we can then rename multilineHandler to something more generic?

Reply:
Yes, that could work.

Reply:
Should I rename the class to something like 'InputHandler' now as we have moved methods to process commands inside?

Reply:
Yes, that should work. We can always rename later.